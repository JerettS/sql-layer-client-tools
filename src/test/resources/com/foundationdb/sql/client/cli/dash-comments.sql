SELECT 1;
SELECT 2;-- foo
SELECT 3; -- bar
SELECT 4 -- zap
;
-- pop
-- zip
SELECT 5;
