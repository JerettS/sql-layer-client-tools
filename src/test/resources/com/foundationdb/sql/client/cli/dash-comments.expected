_SQL_COL_1 
------------
          1 
(1 row)

 _SQL_COL_1 
------------
          2 
(1 row)

 _SQL_COL_1 
------------
          3 
(1 row)

 _SQL_COL_1 
------------
          4 
(1 row)

 _SQL_COL_1 
------------
          5 
(1 row)

   _SQL_COL_1    
-----------------
 foo--bar -- zap 
(1 row)

